src/app/content/hooks/locationChange/resolveContent.ts

```diff
@@ -96,16 +96,12 @@ export const resolveBookReference = async(
     throw new BookNotFoundError(`Could not resolve uuid for slug: ${bookSlug}`);
   }
 
-  const bookVersion = 'version' in match.params.book
-    ? match.params.book.version === 'latest'
```

Comment:
removing this breaks specifying a book@version in the url that is not in the config right? we need that